client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -699,7 +699,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                     allInvestigators={getFilteredUsersOfCurrentCounty()}
                     checkedRowsIds={checkedRowsIds}
                     allDesks={allDesks}
-                    onClose={() => setCheckedRowsIds([])}
```

Comment:
I think onClose is still an appropriate name for this prop.

Reply:
I thought to keep the name close to that, but @ItayBenMoshe insisted on the new name